### DISCUSSION TITLE
'http_403': 1, 'req_err': 1

### DISCUSSION THREAD

**Discussion by gs3009** [kubilus1/autoortho#487] [General]
Hi

I have installed 0.7.2 but get this error in my log, and the UI closes.

**'http_403': 1, 'req_err': 1**

All scenery is installed.

V ).4.4 works fine.

Any help appreciated.

Win 10

Thanks

**Comment by gs3009** [comment]
Solved!

**Reply by katac** [reply]
> Solved!

how ? 
I ahve the same error

**Comment by Jeff2122** [comment]
Can u please post your solution? Thanks.

Whether it was a Windows glitch or if it was a mistake on your part, it may help others.

**Comment by lowpass777** [comment]
Please tell the solution

**Comment by mercster** [comment]
LOL, gotta love jerkoffs like that.  "HEY I SOLVED THE PROBLEM NEVERMIND" then disappears into the aether.
